src/Umbraco.Core/Factories/IMachineInfoFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Umbraco.Cms.Core.Factories;
+
+/// <summary>
+/// Fetches information of the host machine
+/// </summary>
+public interface IMachineInfoFactory
+{
+    /// <summary>
+    /// Fetches the name of the Host Machine for identification
+    /// </summary>
+    /// <returns>A name of the host machine.</returns>
+    public string GetMachineName();
```

Comment:
Let's use a different name for this method. `GetMachineName` encodes that what's returned is the name of the machine; however, what we want, functionally, is just a unique identifier of the machine. The fact that it is currently the machine name is an implementation detail. Maybe `GetMachineIdentifier`? :thinking: